### ISSUE TITLE
Suggestion: prompt for Server on HS startup

### ISSUE THREAD

**Issue by GiggliG** [Epix37/Hearthstone-Deck-Tracker#2181]
I have found HDT very erratic when it comes to recording the server you're playing on. It mostly gets it right, but still gets it wrong pretty often. 

My suggestion: include an option that, when HDT detects you are starting up HS, it pops up a message asking which server you are playing on. You select it, then it automatically uses that server for all games in the current session.

**Comment by Epix37** [comment]
Do you have logs of it failing to detect the new region when (re)starting HS?

**Comment by GiggliG** [comment]
If it happens again and I notice it immediately, I'll save the logs. But I tend not to notice it when it happens, but only notice it later. For instance, this suggestion was prompted by looking at the stats of a deck I've only played on the EU server, and I noticed the first 2 games, played at the start of february, were listed as being the asia server.

(I checked the replays to make sure the games were assigned to the correrct deck, and confirmed they couldnt have been played on Asia because i dont have the cards there.)

What I suspect may be happening is that it is using the last detected server, because I probably opened asia up earlier that day to check quests.

Also, it's a pain having to manually correct these one by one. It would be great to have an ability to edit games in bulk - e.g. select three games, change server in all 3 at once.
